data_bank/mine_database.py

```diff
@@ -91,3 +92,69 @@ def get_blocks_between(self, begin_timestamp=0, end_timestamp=datetime_string_to
                                            "" if number_of_blocks is None else "LIMIT {0}".format(number_of_blocks))
         blocks = self.execute_select(select_sql_query=sql_query)
         return [(b[0], b[1],) for b in blocks]
+
+    def get_slushpool_info(self, begin_timestamp=0, end_timestamp=datetime_string_to_timestamp(datetime_string=None),
```

Comment:
I suggest change the method name to `get_pool_info` and add a pool name argument whose default is slushpool name. This method can be easily refactored to be used for any pool.